### ISSUE TITLE
Server doesn't start if LE is down/unavailable

### ISSUE THREAD

**Issue by mappu** [dkumor/acmewrapper#13]
Let's Encrypt had some downtime on the 2015-05-19.

This project does always check in with LE during initialisation (`acmewrapper.New` calls `w.initACME`), if LE is down, then `New` returns an error.

Really it should instead use the cached certificate files on disk and, retry the authentication parts of the connection with LE only when the certificate needs updating.

**Comment by dkumor** [comment]
Agreed! It should always start if the cached certificates are not expired.

**Comment by dkumor** [comment]
Unfortunately, I am currently travelling, and I forgot my private key at home, which means that I will not be able to commit a fix until I am back on June 4. In the mean time, I can accept a pull request if you think you can fix this issue!
